medical/models/medical_patient.py

```diff
@@ -57,6 +57,12 @@ def _compute_age(self):
             years_months_days = _('No DoB !')
         self.age = years_months_days
 
+    @api.one
+    def action_invalidate(self):
+        self.active = False
+        self.partner_id.active = False
+        self.dod = fields.Datetime.now()
```

Comment:
INO invalid doesn't mean deceased.  I can understand the need to easily disable a patient and the related partner. The motivations can me be multiple:
* A double entry for the same patient,
* The patient moved to another country,
* ...
Moreover, it's not because the patient is deceased that the all the pending administrative tasks (invoices, ...) are completed. In a lot of cases, you'll need to access to the patient information during a certain period after his death.